src/Symfony/Component/Console/Command/Command.php

```diff
@@ -100,6 +100,10 @@ public function __construct(?string $name = null)
             $this->setDescription(static::getDefaultDescription() ?? '');
         }
 
+        if ('' === $this->help && $attributes = (new \ReflectionClass(static::class))->getAttributes(AsCommand::class)) {
+            $this->setHelp($attributes[0]->newInstance()->help ?? '');
+        }
```

Comment:
we should either add addDefaultHelp or deprecate getDefaultName/Description to be consistent

Reply:
The name and description static methods remain useful for lazy commands, but this approach might not be ideal for `help`. Additionally, introducing a new `getDefaultHelp()` method seems redundant since it's already possible to override `getHelp()` when customization is needed.

Reply:
would you prefer adding a private static method for `getDefaultHelp()` purely for the sake of consistency?